src/MongoDB.Driver.Core/Core/Connections/KeepAliveValues.cs

```diff
@@ -33,7 +33,7 @@ public byte[] ToBytes()
             var bytes = new byte[12];
             BinaryPrimitives.WriteUInt32LittleEndian(new Span<byte>(bytes, 0, 4), OnOff);
             BinaryPrimitives.WriteUInt32LittleEndian(new Span<byte>(bytes, 0, 4), KeepAliveTime);
-            BinaryPrimitives.WriteUInt32LittleEndian(new Span<byte>(bytes, 0, 4), KeepAliveInterval);
+            BinaryPrimitives.WriteUInt32LittleEndian(new Span<byte>(bytes, 0, 8), KeepAliveInterval);
```

Comment:
Similarly this should be:
`BinaryPrimitives.WriteUInt32LittleEndian(new Span<byte>(bytes, 8, 4), KeepAliveTime);`